pkg/kudoctl/cmd/install/install.go

```diff
@@ -36,6 +42,26 @@ func CmdErrorProcessor(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func validateInstallParameters() error {
```

Comment:
Right now, this func returns at the first error. Do we want to capture these errors as a group and report them all? We may want a grouped error package here.

I could be convinced that's a later refactoring as well.

Reply:
That shouldn't be that hard. Let me give it a short try